web/src/engine/websites/decorators/MangaReaderCMS.ts

```diff
@@ -0,0 +1,189 @@
+import { FetchRequest, FetchCSS } from '../../FetchProvider';
+import { type MangaScraper, Manga, type Chapter, type Page, type MangaPlugin } from '../../providers/MangaPlugin';
+import * as Common from './Common';
+
+function ChapterInfoExtractor(element : HTMLElement) {
+    const anchor: HTMLAnchorElement = element.nodeName.toLowerCase() === 'a' ? element as HTMLAnchorElement : element.querySelector('a');
```

Comment:
Consider to use `element instanceof HTMLAnchorElement` for type detection

Reply:
Is it safe to assume `anchor` is always set to a valid value or should there be any error handling in case of `undefined`?

Reply:
1) OK (i merely pasted the old code )
2) anchor should always be defined for a given website. I think its like this because there maybe case where we have to extract more than the anchor infos so we use the parent control as selector. 

I tested all the (not dead) websites we had, may mangas and it seems consistent.